scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -275,6 +308,11 @@ private void start(final InetSocketAddress bindAddress) throws IOException {
 		timer = Executors.newSingleThreadScheduledExecutor(
 				new DaemonThreadFactory("DTLS RetransmitTask-", NamedThreadFactory.SCANDIUM_THREAD_GROUP));
 
+		if (executor == null) {
+			// use a thread pool with max 6 * #CPU cores
+			executor = new StripedExecutorService(Runtime.getRuntime().availableProcessors() * 6);
```

Comment:
Just an *educated guess* ;-) Should be more than the number of cores because we still expect these threads to block so it should be a *pool* instead of just a *small set*.
Would you propose another default value?